components/dashboard/src/settings/Teams.tsx

```diff
@@ -0,0 +1,767 @@
+        const plan = getPlan();
+        const expectedPrice = quantity * plan.pricePerMonth;
+        setExpectedPrice(`${expectedPrice}${Currency.getSymbol(plan.currency)}`);
+    }, [quantity])
+
+    const getPlan = () => {
+        return Plans.getAvailableTeamPlans().filter(p => p.chargebeeId === props.sub.planId)[0];
+    }
+
+    return (<Modal visible={true} onClose={props.onClose}>
+        <h3 className="pb-2">Add Members</h3>
+        <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-4">
+            <p className="pb-4 text-gray-500 text-base">Add members to the team.</p>
+
+            <div className="flex flex-col space-y-2 pb-4">
+                <label htmlFor="quantity" className="font-medium">Members</label>
+                <select name="quantity" value={quantity} className="rounded-md w-full border-2 border-gray-400"
+                    onChange={(e) => setQuantity(e.target.value as any)}>
+                    {Array(20).fill(1).map((_, index) => index + 1).map(n => (
+                        <option key={`quantity-${n}`} value={n}>{n}</option>
+                    ))}
+                </select>
+            </div>
+
+            <div className="flex rounded-md bg-gitpod-kumquat-light p-3">
+                <img className="w-4 h-4 mx-2 my-auto" src={exclamation} />
+                <span className="text-red-600">Total: {expectedPrice} per month</span>
+            </div>
+
+        </div>
+        <div className="flex justify-end mt-6">
+        <button className={"ml-2"} onClick={() => props.onBuy(getPlan(), quantity, props.sub)}>Continue to Billing</button>
+        </div>
+    </Modal>);
+}
+
+function NewTeamModal(props: {
+    types: string[],
+    onBuy: (plan: Plan, quantity: number) => void,
+    onClose: () => void,
+}) {
+
+    const [type, setType] = useState<string>(props.types[0]);
+    const [quantity, setQuantity] = useState<number>(5);
+
+    const [expectedPrice, setExpectedPrice] = useState<string>("");
+
+    useEffect(() => {
+        const plan = getPlan(type);
+        const expectedPrice = quantity * plan.pricePerMonth;
+        setExpectedPrice(`${expectedPrice}${Currency.getSymbol(plan.currency)}`);
+    }, [type, quantity])
+
+    const getPlan = (type: string) => {
+        return Plans.getAvailableTeamPlans().filter(p => p.type === type)[0];
+    }
+
+    const teamTypeLabel = (type: string) => {
+        return getPlan(type)?.name;
+    }
+
+    return (<Modal visible={true} onClose={props.onClose}>
+        <h3 className="pb-2">New Team</h3>
+        <div className="border-t border-b border-gray-200 mt-2 -mx-6 px-6 py-4 space-y-2">
+            <p className="pb-4 text-gray-500 text-base">Create a team and add team members.</p>
+
+            <div className="flex flex-col space-y-2">
+                <label htmlFor="type" className="font-medium">Team</label>
+                <select name="type" value={type} className="rounded-md w-full border-2 border-gray-400"
```

Comment:
**issue**: We've changed the style of the text field inputs, see [`index.css`](https://github.com/gitpod-io/gitpod/blob/main/components/dashboard/src/index.css#L53-L67) (`L53`-`L67`). Decreased border size, changed border color, etc. Could we use the same style for the select input?